Real Estate/1 sketch.sql

```diff
@@ -0,0 +1,28 @@
+/*
+
+House
```

Comment:
I read the sketch and was going to make some comments (about bedrooms or bathrooms being zero), but then I saw that the table wasnt actually written exactly following your sketch. Not a big deal, but they should try to be consistent with each other.